cabby/data/wikigeo.py

```diff
@@ -0,0 +1,74 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''Basic classes and functions for Wikigeo items.'''
+
+import re
+from typing import Text
+from shapely.geometry.point import Point
+
+import attr
+
+
+@attr.s
+class Entity:
```

Comment:
Let's name this WikigeoEntity to keep it distinct from wikidata.item.Entity. (We should perhaps rename the latter too.)